sklearn/tests/test_base.py

```diff
@@ -486,3 +486,74 @@ def test_tag_inheritance():
     diamond_tag_est = DiamondOverwriteTag()
     with pytest.raises(TypeError, match="Inconsistent values for tag"):
         diamond_tag_est._get_tags()
+
+
+def test_sub_estimator_consistency():
```

Comment:
this test is pretty irrelevant to the whole thing, not sure it should be included here.
It basically ensures that all fitted sub-estimators in meta-estimators are called
'estimator_', 'base_estimator_', 'one_hot_encoder_' or 'estimators_'.

Reply:
+1 for a separate PR, and adding the fact that this is expected from meta-estimators to the related docs?

Reply:
so you want to add an attribute to all meta-estimators? I'm not entirely convinced that's the right thing to do but also not very opposed.
Basically my implementation would be making the current ``get_sub_estimators`` a property of the ``MetaEstimatorMixin``.